template.yaml

```
@@ -27,7 +27,7 @@ Globals:
         API_TOKEN_NAME: !Ref ApiTokenName
         DC_API_ENDPOINT: !Ref DcApiEndpoint
         DC_URL: !Ref DcUrl
-        DEFAULT_SEARCH_SIZE: "100"
+        DEFAULT_SEARCH_SIZE: "4"
```

```suggestion
        DEFAULT_SEARCH_SIZE: "100"
```

I think this was left in here by mistake.